bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/AbstractTestFilters.java

```diff
@@ -2180,6 +2223,24 @@ protected Table addDataForTesting(String rowKeyPrefix, byte[] qualA)
     return table;
   }
 
+  private void assertRowKeysWithRegex(Table table, String[] rowRegEx,
+      String rowPrefix, String[] rowKeys, int[] expected) throws IOException {
+    FilterList filtersList = new FilterList(FilterList.Operator.MUST_PASS_ONE);
+    for (String condition : rowRegEx) {
+      filtersList.addFilter(new RowFilter(CompareOp.EQUAL,
+          new RegexStringComparator(rowPrefix + condition)));
+    }
+    Scan scan = new Scan();
+    scan.setFilter(filtersList);
+    try (ResultScanner scanner = table.getScanner(scan)) {
+      Result[] results = scanner.next(10);
```

Comment:
`scanner.next(rowKeys.length)` might be better than `scanner.next(10)`

Reply:
Thanks for the suggestion, Address the same.